core/src/OpenMPTarget/Kokkos_OpenMPTarget_Reducer.hpp

```diff
@@ -236,12 +236,12 @@ struct OpenMPTargetReducerWrapper<MinLoc<Scalar, Index, Space>> {
   // Required
   KOKKOS_INLINE_FUNCTION
   static void join(value_type& dest, const value_type& src) {
-    if (src.val < dest.val) dest = src;
-  }
-
-  KOKKOS_INLINE_FUNCTION
-  static void join(volatile value_type& dest, const volatile value_type& src) {
-    if (src.val < dest.val) dest = src;
```

Comment:
That's not realli necessary here but I would want to remove the `volatile` overloads anyway.